core/src/main/scala/org/bitcoins/core/gcs/SipHashKey.scala

```diff
@@ -0,0 +1,17 @@
+package org.bitcoins.core.gcs
+import scodec.bits.ByteVector
+case class SipHashKey(bytes: ByteVector) {
```

Comment:
from inside `sbt`, run scalafmt and it will autoformat your code to look like the rest of our codebase :)